File: app/src/main/java/com/cookplan/recipe_grid/RecipeGridView.java
package com.cookplan.recipe_grid;

import com.cookplan.models.Recipe;
import com.cookplan.share.ShareView;

import java.util.List;

/**
 * Created by DariaEfimova on 21.03.17.
 */

public interface RecipeGridView extends ShareView{

    void setErrorToast(String error);

    void setRecipeList(List<Recipe> ingredientList);

    void setEmptyView();
}

File: app/src/main/java/com/cookplan/recipe/grid/RecipeGridView.kt
package com.cookplan.recipe.grid

import com.cookplan.models.Recipe

/**
 * Created by DariaEfimova on 21.03.17.
 */

interface RecipeGridView {

    fun setErrorToast(error: String)

    fun setRecipeList(recipeList: List<Recipe>)

    fun setEmptyView()
}
